services/apps/premium/members_enrichment_worker/src/factory.ts

```
@@ -0,0 +1,28 @@
+        return new EnrichmentServiceProgAI(
+          log,
+          ALSO_USE_EMAIL_IDENTITIES_FOR_ENRICHMENT,
+          ENRICH_EMAIL_IDENTITIES,
+        )
+      case MemberEnrichmentSource.CLEARBIT:
+        return new EnrichmentServiceClearbit(log)
+      default:
+        throw new Error(`Enrichment source ${source} is not supported`)
+    }
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)